client/state/data-layer/wpcom/activity-log/rewind/to/index.js

```diff
@@ -35,6 +40,14 @@ const requestRewind = ( action, payload ) =>
 	);
 
 const requestRestore = ( action ) => requestRewind( action, { types: action.args } );
+
+const requestGranularRestore = ( action ) =>
+	requestRewind( action, {
+		types: 'paths',
+		include_path_list: action.includePaths,
+		...( action.excludePaths ? { exclude_path_list: action.excludePaths } : {} ),
```

Comment:
This change is addressing https://github.com/Automattic/wp-calypso/pull/81326#pullrequestreview-1608716491:

> The restore logs mention one included path and one excluded path. I'm not sure if this is because we're passing an empty excluded path. We may be expected to trim it and only pass excluded paths if they're non-empty. I think functionally it behaves the same, but we should just see if this is a restore side issue or a case we handle here.